perun-wui-profile/src/main/resources/cz/metacentrum/perun/wui/profile/client/resources/PerunProfileTranslation_cs.properties

```diff
@@ -110,3 +110,15 @@ sendingRequest=Odesílání žádosti
 requestedQuota=Požadovaná kvóta
 reason=Důvod
 notSet=Nenastavena
+menuAltPasswords=Alternativní hesla
+description=Popis
+addAltPassword=Přidat alternativní heslo
+delete=Vymazat
+noPasswords=Žádné heslá nebyli nalezeny
+generatedPassword=Vaše vygenerované heslo
+generatePassword=Vygenerujte nové alternativní heslo
+yourAlternativePasswords= Vaše alternativní heslá
+passwordDescription=Popis hesla
+generatedPasswordHint=Prosím, vložte tohle heslo do aplikace okamžite. Heslo nebude ukázné opět.
```

Comment:
I'm unsure about this one. Does the user know that password is supposed to be "application" or "device" password for his einfra account ? Maybe more generic text will suit the use-case better. Like ``Please copy the password now. It won't be shown again.`` (Prosím zkopírujte si heslo nyní. Nebude vícekrát zobrazeno.)